apps/src/p5lab/poetry/PoetryLibrary.js

```diff
@@ -19,7 +20,7 @@ export default class PoetryLibrary extends CoreLibrary {
       endTime: POEM_DURATION
     };
     this.poemState = {
-      ...getStore().getState().poetry.selectedPoem,
+      ..._.cloneDeep(getStore().getState().poetry.selectedPoem),
```

Comment:
how was this previously modifying redux state?

Reply:
In `addLine`, we push to the `lines` array in `poemState`:
https://github.com/code-dot-org/code-dot-org/blob/4f9837a2a7b0795f8c427d5e0f4a7901292e5281/apps/src/p5lab/poetry/PoetryLibrary.js#L90

The spread operator doesn't make a copy of the `lines` array, so this ended up modifying the actual object in the redux state, which was not desirable.